packages/story-editor/src/components/videoTrim/provider.js

```diff
@@ -64,6 +65,9 @@ function VideoTrimProvider({ children }) {
       start: formatMsToHMS(startOffset),
       end: formatMsToHMS(endOffset),
     });
+    trackEvent('video_trim', {
```

Comment:
@swissspidy I've added some track events, not sure if we have some specific naming we follow here, let me know if that looks ok to you.

Reply:
Thanks.

We use extra params like `name` only to pass additional information for an event, e.g. `search_term` for a `search` event.

Here I'd do something more like this:

```js
trackEvent('video_trim', {
  original_length: resource.length
  new_length: lengthInSeconds,
  start_offset: startOffset,
  end_offset: endOffset,
}
```